### PR TITLE
Add button to launch visualization of entire module dependency tree

### PR DESCRIPTION
**Pull request by paulirish**
[anvaka/npmgraph.an](https://github.com/anvaka/npmgraph.an) provides a nice visualization of the entire dep tree.

This PR adds a button into the injected DOM so you can easily click to view this tree:

image

### CODE REVIEW

File: extension/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-$(() => {
+$().ready(() => {
```
Comment (bfred-it) [inline extension/index.js]: Is this necessary?

File: extension/index.js [hunk 2]
```diff
@@ -49,4 +54,13 @@ $(() => {
     }
   }
 
-})
+  function viewDepsViz(e) {
+    const npmPkgName = e.data.pkg.name;
+    const windowFeatures = "resizable,scrollbars,status";
+
+    var otherWindow = window.open(`http://npm.anvaka.com/#/view/2d/${npmPkgName}`, "npmanvaka", windowFeatures);
```
Comment (bfred-it) [inline extension/index.js]: I'd be ok with this just being a simple `<a href>` instead, like all other links

File: extension/index.js (hunk 1 above)
Comment (paulirish) [inline extension/index.js]: not a functional change, but I prefer being clear with how things are being initialized. 

tbh, at first glance i didn't expect it to be jquery.

File: extension/index.js (hunk 2 above)
Comment (paulirish) [inline extension/index.js]: sure. that works for me.

Comment (paulirish) [inline extension/index.js]: done!

**Comment by paulirish** [comment]
Anything else I can do on this?

**Comment by bfred-it** [comment]
LGTM but we have to wait for @zeke 😅

**Comment by zeke** [comment]
This looks pretty cool. Sorry I haven't had time to maintain this module lately. @bfred-it I just added you as a collaborator.

Do you know if there's a way I can add you as an admin on the chrome web store? I don't see an option for that in the UI. Maybe @paulirish knows?

**Comment by paulirish** [comment]
yup!

https://developer.chrome.com/webstore/publish#set-up-group-publishing

Basically: look at the bottom of https://chrome.google.com/webstore/developer/dashboard
you'll want to make a google group that has the people in it.
then you can select that google group
and then in the dashboard UI you can transfer the extensions into the group.

**Comment by paulirish** [comment]
@zeke if you merge #13 i can rebase this PR

**Comment by paulirish** [comment]
thx. rebased

**Comment by bfred-it** [comment]
Does group publishing still work? The [help page](http://web.archive.org/web/20131109081503/https://support.google.com/chrome_webstore/answer/3052863?p=group_publishing) disappeared and I don't see any groups in my account.

Once the autodeployment is setup though this won't matter. 

@zeke Did you have any luck with the OAuth process?

**Comment by zeke** [comment]
Move the extension to the new group. You two should be able to publish now.

screen shot 2016-09-20 at 10 20 49 pm

>  Did you have any luck with the OAuth process?

I got stuck at some point. If you are unable to publish with the new setup, let me know and I'll give it another go. Thanks.

**Comment by bfred-it** [comment]
I think it's because my account has no publishing rights:

> [**Publishing/Things to note**](https://developer.chrome.com/webstore/publish#things-to-note)
> * Each member of the group must pay the developer fee.

I sent you a request to join the group with my other account, which is what I use to publish mine with

**Comment by zeke** [comment]
> I sent you a request to join the group

You should be in! Let me know.

**Comment by bfred-it** [comment]
It worked! Thanks :)

**Pull request merged**